### PR TITLE
initial dom4j port of q40

### PR DESCRIPTION
**Pull request by baranowb**
note: q40 has been stripped of template def, but it still includes it in resources, is this on purpose?

### CODE REVIEW

**Comment by baranowb** [comment]
This one also does not use template.

**Comment by pmuir** [comment]
No, you could send a pull req to remove it.

File: xml-dom4j/pom.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0"?>
```
Comment (pmuir) [inline xml-dom4j/pom.xml:1]: Could you indent by 4 spaces, just like Java files in all files?

File: xml-dom4j/pom.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.jboss.as.quickstarts</groupId>
+	<artifactId>jboss-as-xml-dom4j</artifactId>
+	<version>7.0.2-SNAPSHOT</version>
```
Comment (pmuir) [inline xml-dom4j/pom.xml:10]: Version is now 7.1.1-SNAPSHOT :-D

**Comment by pmuir** [comment]
I assume the only difference between this and xml-jaxp is the in DOM4JXMLParser?

**Comment by baranowb** [comment]
Yeah, thought thats the point :) ? Like hibernate demos.

**Comment by pmuir** [comment]
Yes. But you confirming that saves me checking it :-)

File: xml-dom4j/pom.xml (hunk 2 above)
Comment (baranowb) [inline xml-dom4j/pom.xml:10]: hmm seems like other quickstarts need update also :)

**Comment by pmuir** [comment]
I'll make the updates when I do the merge.

**Comment by pmuir** [comment]
Merged

**Pull request closed without merging**